taroscript/send_test.go

```diff
@@ -2095,119 +1805,101 @@ func TestAddressValidInput(t *testing.T) {
 	}
 }
 
+func addrToDesc(addr address.Taro) *taroscript.RecipientDescriptor {
+	return &taroscript.RecipientDescriptor{
```

Comment:
Also dropping script key here.

Reply:
Yep, that's on purpose. I hope with the new name it's a bit more clear why we don't need the script key.